spec/presenters/tree_builder_containers_spec.rb

```diff
@@ -0,0 +1,36 @@
+describe TreeBuilderContainers do
+  let(:tag) { FactoryGirl.create(:tag, :name => "/managed/department/accounting") }
+  let(:role) { FactoryGirl.create(:miq_user_role, :name => "EvmRole-operator") }
+  let(:group) { FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Group 1") }
+  let(:user) { FactoryGirl.create(:user, :userid => 'User 1', :miq_groups => [group]) }
+
+  before do
+    MiqRegion.seed
+    EvmSpecHelper.local_miq_server
+
+    @tagged_container = FactoryGirl.create(:container, :tags => [tag])
+    @untagged_container = FactoryGirl.create(:container)
+
+    allow_any_instance_of(TreeNodeBuilder).to receive(:generic_node)
+    allow_any_instance_of(TreeBuilderContainers).to receive(:x_build_single_node).and_return({})
+
+    login_as user
+  end
+
+  describe "x_get_tree_roots" do
```

Comment:
We should add the `#` prefix to indicate an instance method

`describe "#x_get_tree_roots" do`